pkg/builder/replacement_entity_meta_data_builder.go

```diff
@@ -4,6 +4,20 @@ import (
 	"github.com/turbonomic/turbo-go-sdk/pkg/proto"
 )
 
+const (
+	PropertyUsed = "used"
+	PropertyCapacity = "capacity"
+	PropertyResizable = "resizable"
+	PropertyLimit = "limit"
+	PropertyPeak = "peak"
+	PropertyComputeUsed = "computeUsed"
+	PropertyReservation = "reservation"
+)
+
+var (
+	defaultPropertyNames = []string{PropertyUsed, PropertyResizable, PropertyComputeUsed, PropertyCapacity}
```

Comment:
Create different default properties list for sold and bought commodities. Capacity is not relevant for bought commodities.